src/OpenTK.Mathematics/Matrix/Matrix3.cs

```diff
@@ -354,7 +349,7 @@ public Matrix3 Inverted()
         /// <summary>
         /// Returns a copy of this Matrix3 without scale.
         /// </summary>
-        /// <returns>The matrix.</returns>
+        /// <returns>The matrix without scaling.</returns>
```

Comment:
What aboout these ones, because The return value comment here does not specify that it is a copy. Perhaps we should mark such functions as "Pure" as well?

Reply:
Marking them as pure is a good idea. In general, JetBrains annotations should be used to annotate contracts, purity, and nullability.

Reply:
I left this be at first. And I think I will do pure functions in a separate commit(perhaps even separate PR?).
Question is do we want to have the annotations in the public API as well? Because Jetbrains recommends that if we to, to include it directly into the SourceCode instead of using the nuget package.

Reply:
We'll be using Fody to generate external annotations, which is what we're already doing in the other projects.